core/src/main/resources/audio/article/index.scala.html

```diff
@@ -0,0 +1,46 @@
+@(atom: com.gu.contentatom.thrift.Atom
+, data: com.gu.contentatom.thrift.atom.audio.AudioAtom
+)(implicit conf: com.gu.contentatom.renderer.ArticleConfiguration)
+
+@fragments.atoms.html.audio(
+    atom.id,
+    className = "podcast",
+    label = "Today in Focus",   // TODO: get from data
+    headline = atom.title.getOrElse("")
+){
+    <audio
+    class="atom--audio__player-element"
+    duration="@data.duration"
+    src="@data.trackUrl"
+    preload="metadata"
+    data-media-id="@data.contentId"
+    data-title="@data.kicker"
+    data-component="@data.contentId"
```

Comment:
data-component should be something like: `inarticle audio`
This is what we use on the main player: `data-component="main audio"`

Reply:
Good point. Maybe ask Stu for confirmation.